xbmc/application/ApplicationVolumeHandling.h

```diff
@@ -24,8 +24,8 @@ class CApplicationVolumeHandling
   // times per second (saves doing settings lookup)
   struct ReplayGainSettings
   {
-    int iPreAmp;
-    int iNoGainPreAmp;
+    float iPreAmp;
```

Comment:
i saw now,
suggest to rename `iPreAmp` to `m_preAmp` to prevent confusions
also for `iNoGainPreAmp`

Reply:
I changed all the members of the `ReplayGainSettings` struct so they are in line with the current coding standard.